.pipelines/ci.yml

```diff
@@ -5,7 +5,7 @@ trigger:
     - master
   paths:
     exclude:
-    - docs/*
+    - docs/
```

Comment:
It should probably be a recursive wildcard instead: `**`. Found some docs on it:

* https://docs.microsoft.com/en-us/azure/devops/release-notes/2021/sprint-192-update#support-for-wild-cards-in-path-filters
* https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml
* https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/file-matching-patterns?view=azure-devops